specs/MultiProfile.md

```diff
@@ -29,6 +29,21 @@ Users can use this interface to get the cookie manager, which is shared by all W
 with this profile. The cookie manager got from profile (CoreWebView2.Profile.CookieManager) is the
 same as that got from CoreWebView2 (CoreWebView2.CookieManager).
 
+Currently, we already have **CoreWebView2Settings** interface to manage password-autosave and
+general-autofill, but it will work not immediately but after the next navigation, and it can only
+apply for current WebView2, which means if we start a new WebView2 using the same profile, all the
+settings are default values and cannot be set from the profile. By adding password-autosave and
+general-autofill management interfaces in profile, we can manage the properties and they will apply
+immediately if we set a new value, and all WebView2s that created with the same profile can share
+the settings, which means if we change password-autosave or general-autofill property in one WebView2,
+the others with the same profile will also work. And these two property is linked with the same
```

Comment:
I don't understand what this part is saying. I mean, I hope the properties in all the WebView2's work according to the documentation. If they didn't work, it'd be a bug, right?

Reply:
It means when create multiple instances of WebView2 using the same profile, they will share the same profile-level properties (containing general-autofill and password-autosave). If change a property in one WebView2's profile object, it will be reflected in all other instances of WebView2 that were created with the same profile.

For example, if one WebView2 instance has password-autosave enabled, any other WebView2 instances created with the same profile will also have password-autosave enabled. Similarly, if one WebView2 instance has general-autofill disabled, any other WebView2 instances created with the same profile will also have general-autofill disabled.